packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/organization/general-information/OrganizationGeneralInformation.page.tsx

```diff
@@ -43,7 +43,7 @@ function GeneralInformation() {
         <OrganizationOptionsTile
           isLicenseActive={!!vcdOrganization?.data?.currentState?.spla}
         />
-        <DataProtectionTile />
+        <DataProtectionTile vcdOrganization={vcdOrganization?.data} />
```

Comment:
vcdOrganization is not optionnal for `DataProtectionTile`. So we need change the props type or protect this call to not have `?` necessity